internal/stores/contracts.go

```diff
@@ -179,3 +194,324 @@ func NewJSONContractStore(dir string) (*JSONContractStore, error) {
 	}
 	return s, nil
 }
+
+type (
+	// SQLContractStore implements the bus.ContractStore interface using SQL as the
+	// persistence backend.
+	SQLContractStore struct {
+		db *gorm.DB
+	}
+
+	dbContractRHPv2 struct {
+		ID         types.FileContractID         `gorm:"primaryKey,type:bytes;serializer:gob;NOT NULL"`
+		Revision   dbFileContractRevision       `gorm:"constraint:OnDelete:CASCADE;foreignKey:ParentID;references:ID;NOT NULL"` //CASCADE to delete revision too
+		Signatures []types.TransactionSignature `gorm:"type:bytes;serializer:gob;NOT NULL"`
```

Comment:
@lukechampine Should this be in the revision table? The signatures update for every revision right?